Release 2025.04.0 (#5379)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/release-notes/2025-okta-identity-engine/index.md

| [Risk Provider and Risk Events APIs are deprecated](#risk-provider-and-risk-events-apis-are-deprecated) | April 2, 2025 |
| [POST requests to authorize endpoint is GA Preview](#post-requests-to-authorize-endpoint-is-ga-preview) | January 8, 2025 |
| [Integration variable limit increase in OIN Submission](#integration-variable-limit-increase-in-oin-submission) | April 2, 2025 |
| [Conditional requests and entity tags are GA in Production](#conditional-requests-and-entity-tags-are-ga-in-production) | January 16, 2025 |
| [New rate limit event type](#new-rate-limit-event-type) | April 2, 2025 |
| [Create dynamic resource sets with conditions is GA in Preview](#create-dynamic-resource-sets-with-conditions-is-ga-in-preview) | November 7, 2024 |
| [Okta account management policy is GA in Production](#okta-account-management-policy-is-ga-in-production) | December 11, 2024 |
| [Okta Sign-In Widget custom template updates](#okta-sign-in-widget-custom-template-updates) | April 2, 2025 |
| [Developer documentation update in 2025.04.0](#developer-documentation-update-in-2025-04-0) | April 2, 2025 |
| [Bugs fixed in 2025.04.0](#bugs-fixed-in-2025-04-0)| April 2, 2025 |

> #### OAuth 2.0 provisioning for Org2Org with Autorotation is self-service EA in Preview

Reviewer: ```suggestion
#### OAuth 2.0 provisioning for Org2Org with auto-rotation is self-service EA in Preview
```

These APIs have been deprecated. Use the [SSF Security Event Tokens API]([https://developer.okta.com/docs/api/openapi/okta-management/management/tag/SSFSecurityEventToken/|https://developer.okta.com/docs/api/openapi/okta-management/management/tag/SSFSecurityEventToken/]) instead to receive security-related events and other data-subject signals. Use the [SSF Receiver API]([https://developer.okta.com/docs/api/openapi/okta-management/management/tag/SSFReceiver/|https://developer.okta.com/docs/api/openapi/okta-management/management/tag/SSFReceiver/]) for third-party security event providers. <!-- OKTA-866092 -->

#### POST requests to authorize endpoint is GA Preview

You can now send user data securely in a POST request body to the `/authorize` endpoint. <!-- OKTA-827104 OAUTH2_AUTHORIZE_WITH_POST -->

#### Integration variable limit increase in OIN Submission

The maximum number of integration variables allowed in OIN submission has increased from three to eight. The apps migrated from OIN Manager with more than eight variables can retain all existing variables but can't add new variables.

#### Conditional requests and entity tags are GA in Production

> You can now use conditional requests and entity tags to tag and check for specific versions of resources. Currently this is only available to use with user profiles. See [Conditional Requests and Entity Tags](https://developer.okta.com/docs/api/#conditional-requests-and-entity-tags).

Reviewer 2: ```suggestion
You can now use conditional requests and entity tags to tag and check for specific versions of resources. Currently this is only available to use with user profiles. See [Conditional Requests and Entity Tags](https://developer.okta.com/docs/api/#conditional-requests-and-entity-tags).  <!-- ETAG_FOR_USERS_API OKTA-822688 -->
```

# Okta Identity Engine API release notes (2025)

## April

### Monthly release 2025.04.0

| Change | Expected in Preview Orgs |
|--------|--------------------------|
| [Domain restrictions on Realms](#domain-restrictions-on-realms) | April 2, 2025 |

> | [OAuth 2.0 provisioning for Org2Org with Auto-Rotation is self-service EA in Preview](#oauth-20-provisioning-for-org2org-with-auto-rotation-is-self-service-ea-in-preview) | April 2, 2025 |

Reviewer: ```suggestion
| [OAuth 2.0 provisioning for Org2Org with Auto-Rotation is EA in Preview](#oauth-20-provisioning-for-org2org-with-auto-rotation-is-self-service-ea-in-preview) | April 2, 2025 |
```

| [New rate limit event type](#new-rate-limit-event-type) | April 2, 2025 |
| [Create dynamic resource sets with conditions is GA in Preview](#create-dynamic-resource-sets-with-conditions-is-ga-in-preview) | November 7, 2024 |
| [Okta account management policy is GA in Production](#okta-account-management-policy-is-ga-in-production) | December 11, 2024 |
| [Okta Sign-In Widget custom template updates](#okta-sign-in-widget-custom-template-updates) | April 2, 2025 |
| [Developer documentation update in 2025.04.0](#developer-documentation-update-in-2025-04-0) | April 2, 2025 |
| [Bugs fixed in 2025.04.0](#bugs-fixed-in-2025-04-0)| April 2, 2025 |

#### Domain restrictions on Realms

You can now limit users to a specific domain in Realms, which adds an extra layer of oversight for realm and partner admins and enforces boundaries between user populations. See the [Realms](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Realm/) and [Realm Assignments](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/RealmAssignment/) APIs. <!-- OKTA-907851 UD_REALMS_PROFILE_WITH_DOMAIN -->

> #### OAuth 2.0 provisioning for Org2Org with Auto-Rotation is self-service EA in Preview

Reviewer: ```suggestion
#### OAuth 2.0 provisioning for Org2Org with Auto-Rotation is EA in Preview
```
